nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/deck_guns.dm

```diff
@@ -587,14 +587,19 @@
 	var/armed = FALSE //Do it do the big boom?
 	var/speed = 0.5 //Needs powder to increase speed.
 
-/obj/item/ship_weapon/ammunition/naval_artillery/attack_hand(mob/user)
-	return FALSE
-
-/obj/item/ship_weapon/ammunition/torpedo/attack_hand(mob/user)
-	return FALSE
+/obj/item/ship_weapon/ammunition/naval_artillery/Initialize() //Sophie: Makes NAC rounds draggable in Zero-G
```

Comment:
Irrelevant to ship PR, please move to separate PR.

Reply:
Separated